Formula/epubcheck.rb

```diff
@@ -4,6 +4,8 @@ class Epubcheck < Formula
   url "https://github.com/w3c/epubcheck/releases/download/v4.2.5/epubcheck-4.2.5.zip"
   sha256 "2f725b7f491bd7e215fe3fb5492d0554207c838c35165cc195963ddcde39c887"
   license "BSD-3-Clause"
+  
+  depends_on "openjdk"
 
   bottle :unneeded
```

Comment:
```suggestion

  bottle :unneeded

  depends_on "openjdk"
```
We should probably use `overrideable_java_home_env` in the `write_jar_script` call.

You used the GitHub UI to do this, didn't you

Reply:
Yes I did 😂